triage/nw_prj_neuron_track_hours/bonita_exporter.py

```
@@ -65,17 +63,24 @@ def _group_locators(shifts: List[BonitaShift]) -> List[object]:
+def _write_month_tab(wb, tab: str, shifts: List[BonitaShift], profile: Dict[str, Any]) -> None:
     Workbook, Alignment, Font, PatternFill, get_column_letter = _require_openpyxl()
     ws = wb.create_sheet(tab)
-    fill = PatternFill("solid", fgColor=_HEADER_FILL)
-    for c, (top, bottom) in enumerate(zip(HEADER_TOP, HEADER_BOTTOM), 1):
+    fill1 = PatternFill("solid", fgColor=profile.get("header_fill_row1", "1F4E78"))
+    fill2 = PatternFill("solid", fgColor=profile.get("header_fill_row2", "5B9BD5"))
+    h1 = profile.get("header_row1") or HEADER_TOP
+    h2 = profile.get("header_row2") or HEADER_BOTTOM
+    asn_fills = profile.get("assignment_fills") or {}
+
+    for c, (top, bottom) in enumerate(zip(h1, h2), 1):
```

Ruff (0.15.15)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)